test/custom.test.tsx

```diff
@@ -1,60 +1,64 @@
-import * as React from 'react';
-import Downshift, { ControllerStateAndHelpers } from '../';
+import * as React from 'react'
+import Downshift, {ControllerStateAndHelpers} from '../'
+
+type Item = string
 
 interface Props {}
 
 interface State {
-    items: Array<any>;
+  items: Array<Item>
 }
 
-const CustomList = ({ isOpen, children }) => (
-    <div className={isOpen ? 'open' : ''}>{children}</div>
-)
+const CustomList: React.StatelessComponent<{isOpen: boolean}> = ({
+  isOpen,
+  children,
+}) => <div className={isOpen ? 'open' : ''}>{children}</div>
 
-const CustomListItem = ({ isSelected, children }) => (
-    <div className={isSelected ? 'selected' : ''}>{children}</div>
-)
+const CustomListItem: React.StatelessComponent<{isSelected: boolean}> = ({
+  isSelected,
+  children,
+}) => <div className={isSelected ? 'selected' : ''}>{children}</div>
 
 export default class App extends React.Component<Props, State> {
-    state: State = {
-        items: ['apple', 'orange', 'carrot'],
-    };
-
-    onChange = (selectedItem: any) => {
-        console.log('selectedItem', selectedItem);
-    };
-
-    render() {
-        const items = this.state.items;
-        const defaultSelectedItem = this.state.items[0];
-
-        return (
-            <Downshift defaultSelectedItem={defaultSelectedItem}>
-                {({
-                    getToggleButtonProps,
```

Comment:
this was changed in master. Should I change it to master version?

Reply:
Yes, sorry, I didn't consider how that would impact your PR. I renamed (and aliased) `getButtonProps` to `getToggleButtonProps`. Sorry about that.